File: Recursion/sumOfDigits.cpp
#include<iostream>
using namespace std;

int sumOfDigits(int num){
    if(num < 1){
        return 0;
    }
    return (num%10)+sumOfDigits(num/10);
}

int main(){
    int number=7225;

    cout<<"Sum of digits of number "<<number<<" is "<<sumOfDigits(number);
}

File: Recursion/sumOfDigits.js
const sumOfDigits=(num)=>{
    if(num<1){
        return 0;
    }
    let sum=num%10
    return sum+sumOfDigits(Math.floor(num/10))
}

let number=7225

let sum=sumOfDigits(number)
console.log(`Sum of digit of number ${number} is ${sum}`)